### ISSUE TITLE
Keyboard navigation between developer tools and vs code window

### ISSUE THREAD

**Issue by leonardder** [microsoft/vscode#97216]
When the chromium developer tools inside visual studio code are open, it is not possible to navigate between the tools and the window of VS Code. I"m not sure how this is done visually, but it might help to add them to the F6 sequence or provide another sequence to switch between the two. I think this  could be really helpful for keyboard dependent users to inspect the DOM tree for accessibility feedback.

Cc @isidorn

**Comment by isidorn** [comment]
@leonardder good point. Though this should be handled by chrome, there should be a keyboard shortcut to close the dev tools.
Unfortunetly we can not integrate F6 to take into account dev tools
@deepak1556 @bpasero do you know of a keybaord shortcut to quit chrome dev tools on Windows? I tried and could not find one.

**Comment by bpasero** [comment]
@isidorn see https://github.com/microsoft/vscode/blob/ff118340b561fea6184172ffaf3bbf859b21055b/src/vs/workbench/electron-browser/actions/developerActions.ts#L13-L29

**Comment by isidorn** [comment]
@bpasero that toggles developer tools from vscode. But can not do it while the focus is in the chrome dev tools, or am I missing something?

**Comment by bpasero** [comment]
Hm I think `CtrlCmd+Shift+I` always works?

**Comment by isidorn** [comment]
@bpasero just tried that and it does not work both on macOS and Win.

**Comment by bpasero** [comment]
F12 it is.

**Comment by leonardder** [comment]
I'm neither able to escape from them with ctrl+shift+i nor f12 on Windows

**Comment by isidorn** [comment]
Yeah also does not work for me.

**Comment by bpasero** [comment]
Maybe open shortcuts panel in devtools to find out more:

![image](https://user-images.githubusercontent.com/900690/81401193-18930480-912f-11ea-9ac8-9a1f35c885e2.png)

**Comment by isidorn** [comment]
@bpasero thanks

Ok so looked a bit more, and we only assign the keybdinig when running out of source.
@leonardder thus I suggest the following: File > preferences > keyboard shortucts
Find the command "Developer: Toggle Developer Tools" and invoke context menu -> change when clause
Delete the `when`  caluse to be empty.
Now you will be able to simply `Ctrl+Shift+I` to toggle it.

Thus closing this as I think this solution might be ok for now.
